src/components/icons/platform/index.jsx

```diff
@@ -292,21 +326,48 @@ export const PlatformLinks = Object.keys(PlatformLinkIcon).map(platform => {
   }
 })
 
-export const Links = (links, content, isPreviewMobile) =>
-  links.list
-    .filter(({ platform, url }) => !!PlatformLinkIcon[platform])
-    .map(link => (
-      <PlatformLink
-        key={link.platform}
-        border={links.border}
-        circle={links.circle}
-        square={links.square}
-        size={links.size}
-        color={content.color}
-        colorAccent={content.colorAccent}
-        colorBackground={content.colorBackground}
-        colorBackgroundAccent={content.colorBackgroundAccent}
-        isPreviewMobile={isPreviewMobile}
-        {...link}
-      />
-    ))
+export const Links = (links, content, isPreviewMobile, Modal) => {
+  const [modalData, setModalData] = useState({
+    show: false,
+    label: 'text',
+    content: '',
+  })
+
+  return (
+    <Fragment>
+      {Modal && (
```

Comment:
This is what you asked, right? Because I do not understand why you should ask for it if you import it on top.